lib/semian/adapter.rb

```diff
@@ -53,7 +59,11 @@ def semian_options
       return @semian_options if defined? @semian_options
 
       options = raw_semian_options
-      @semian_options = options && options.map { |k, v| [k.to_sym, v] }.to_h
+
+      symbolized_options = options && options.map { |k, v| [k.to_sym, v] }.to_h
```

Comment:
Would it better to symbolize Hash keys on initialization part for all adapters or HTTP one?
My goal here to have less operations.

Probably it is a good task for a separate PR.